src/components/form-input-field/input-field.styles.ts

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components";
```

Comment:
Create a components folder in the auth directory and move this there, this component would used by auth pages alone.